src/lib/utility/logging/logging.h

```diff
@@ -5,32 +5,38 @@
 
 #include "utilityString.h"
 
+// Macro to strip the prefix from __FILE__
+#define RELATIVE_FILE_PATH (__FILE__ + SOURCE_PATH_PREFIX_LEN)
+
```

Comment:
Please use constexpr

Reply:
Redundant

Reply:
The macro add a lot of warnings I replaced it with the following
`constexpr auto RelativeFilePath = std::next(__FILE__, static_cast<long>(SOURCE_PATH_PREFIX_LEN));`